src/seedsigner/views/seed_views.py

```diff
@@ -193,20 +194,44 @@ def run(self):
         
         elif button_data[selected_menu_num] == self.TYPE_12WORD:
             self.controller.storage.init_pending_mnemonic(num_words=12)
-            return Destination(SeedMnemonicEntryView)
+            if wordlist_language_code=="en": 
+                return Destination(SeedMnemonicEntryView)
+            else:
+                return Destination(SeedWordlistLanguageWarningView)
 
         elif button_data[selected_menu_num] == self.TYPE_24WORD:
             self.controller.storage.init_pending_mnemonic(num_words=24)
-            return Destination(SeedMnemonicEntryView)
+            if wordlist_language_code=="en": 
+                return Destination(SeedMnemonicEntryView)
+            else:
+                return Destination(SeedWordlistLanguageWarningView)
 
         elif button_data[selected_menu_num] == self.TYPE_ELECTRUM:
             return Destination(SeedElectrumMnemonicStartView)
 
         elif button_data[selected_menu_num] == self.CREATE:
             from .tools_views import ToolsMenuView
             return Destination(ToolsMenuView)
+        
+class SeedWordlistLanguageWarningView(View): 
+    CONTINUE = ButtonOption("Continue", button_label_color="red")
+    BACK = ButtonOption("Go Back")
+    def __init__(self): 
+        super().__init__();
+
+    def run(self): 
+        button_data = [self.CONTINUE, self.BACK]
+        ret = self.run_screen(
+            seed_screens.SeedWordsLanguageWarningScreen, 
+            title=_("Non-English wordlist warning"),
```

Comment:
My fault for not finishing our UX style guide, but titles should be in "title case", meaning that every word's first letter is capitalized.

Buttons (see your `BACK` button) should be in "sentence case" (only first letter of first word is capitalized).

Our UX definitely is not yet standardized to these guidelines, but we will make an effort to do so soon-ish.